cpp/src/link_analysis/pagerank_1D.cuh

```diff
@@ -90,17 +90,17 @@ int pagerank(raft::handle_t const &handle,
              const double tolerance      = 1e-5)
 {
   // null pointers check
-  CUGRAPH_EXPECTS(G.offsets != nullptr, "Invalid API parameter - offsets is null");
-  CUGRAPH_EXPECTS(G.indices != nullptr, "Invalid API parameter - indidices is null");
+  CUGRAPH_EXPECTS(G.offsets != nullptr, "Invalid input argument - offsets is null");

```

Comment:
We have an issue that all error messages start with "Invalid API"  but main of the error checks are not related to parameters that the User specifies.  We need to differentiate between User, System, and Our errors rather than just blaming the user